src/components/EditableField.tsx

```diff
@@ -27,12 +51,13 @@ const EditableField = ({ label, initialValue, type, onSave }: Props) => {
         <h2 className="text-lg m-2">
           Edit your <span className="lowercase"> {label} </span>
         </h2>
-        <div className="p-1 ">
-          <form className="flex justify-betwee flex-col" onSubmit={onSubmit}>
-            <div className="flex border border-black border-solid mb-2 ">
+        <div className="p-1 w-[350px] sm:w-[600px]">
```

Comment:
This looks like we're setting a fixed width to this component. Does this mean that - for example - if I want to use this component in another page but I want it to be 100% of the screen, I cannot do that?